Added information and detection for Firefox ESR

scottnotpat opened the pull request:
Adding detection and information about Firefox ESR (Extended Support Release).

scottnotpat committed:
Add support for Firefox ESR mitigation detection

scottnotpat committed:
Updated Readme.md to add info about Firefox ESR

vrdse commented:
@scottnotpat , thank you for the pull request. From my unterstanding, an outdated version like `53.0.1` would return `true`. Could you update your pull request, to adress that issue?

scottnotpat committed:
Updated scripts to detect ESR

Updated scripts to detect ESR and prevent detection of older versions of the Firefox Rapid Release

scottnotpat commented:
Fixed the pull request to address the issue with older Firefox Rapid Release versions being detected as patched.

vrdse reviewed (approved):
lgtm
